charts/aws-efs-csi-driver/values.yaml

```diff
@@ -101,6 +103,8 @@ controller:
     #  eks.amazonaws.com/role-arn: arn:aws:iam::111122223333:role/efs-csi-role
   healthPort: 9909
   regionalStsEndpoints: false
+  # Pod Disruption Budget
+  pdb: {}
```

Comment:
What about enabling this as default from the chart similar to `ebs` controller,
https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/50f6b2e68446b221bde01fbd3aca190ea9a0e569/charts/aws-ebs-csi-driver/values.yaml#L250-L254
<img width="875" alt="image" src="https://github.com/user-attachments/assets/3799ff49-ae9d-4876-a55a-8206946b3e8d">

Reply:
Currently, this feature doesn't exist. In my opinion, it would be best to disable it by default and only enable it by default once it is well-received by the users.